src/components/sub/menu/menu-bar.jsx

```diff
@@ -1,134 +1,145 @@
-import React, { Fragment } from 'react'
-import MenuItem from './menu-item.jsx'
-import '@/components/sub/menu/menu-bar.scss'
+import React, { Fragment } from "react";
+import MenuItem from "./menu-item.jsx";
+import {
+  RiBold,
+  RiItalic,
+  RiStrikethrough,
+  RiCodeView,
+} from "@remixicon/react";
+import "@/components/sub/menu/menu-bar.scss";
 
 export default ({ editor }) => {
   const items = [
     {
-      icon: 'bold',
-      title: 'Bold',
+      icon: RiBold,
```

Comment:
you have two libraries installed RemixIcon and @remixicon/react

RemixIcon looks like the base font/svg library for the icons which is what you were originally using

@remixicon/react is the same library but is being exported as React Components for your convenience 

Instead of wrangling the SVG fonts (which is possible) @remixicon/react is doing that for us already